include/swift/Runtime/Config.h

```diff
@@ -216,6 +216,7 @@
 // But some of the runtime entries are invoked directly from
 // the foundation. Therefore they should be visible.
 #define SWIFT_RT_ENTRY_VISIBILITY SWIFT_RUNTIME_EXPORT
+#define SWIFT_EXTERN_RT_ENTRY_VISIBILITY SWIFT_EXTERN_RUNTIME_EXPORT
```

Comment:
Thanks. I'll update this PR accordingly.


What about the case of `extern "C"` without `SWIFT_RUNTIME_EXPORT`, such as in https://github.com/apple/swift/blob/adc54c8a4d13fbebfeb68244bac401ef2528d6d0/stdlib/public/runtime/SwiftHashableSupport.h